src/main/java/com/leggeart/legge/Lawsuit.java

```diff
@@ -0,0 +1,122 @@
+package com.leggeart.legge;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.Date;
+
+/**
+ Class responsible for handling Lawsuit object. Contains all data connected
+ to a case.
+
+ @version 0.0.1
+ @author LC
+ */
+@Entity
```

Comment:
@Entity(name = "LAWSUITS")
you don't have to add adnotations @Table if you have already @Entity